bluebrain/deployment/environments/applications_libraries.yaml

```
@@ -57,6 +61,8 @@ spack:
     - py-numpy
     - py-petsc4py
     - py-petsc4py ^petsc+complex+int64+mpi
+    - py-slepc4py ^slepc~arpack
```

Don't you want this version of PETSc for this spec?

```suggestion
    - py-slepc4py ^petsc+int64+mpi ^slepc~arpack
```